src/App.js

```diff
@@ -1,13 +1,31 @@
 import "./App.css";
+import React, { Component } from 'react';
+import { Switch, Route } from 'react-router-dom';
+import NavBar from './Components/NavBar';
+import Movies from './Components/Movies';
+import People from './Components/People';
+import Locations from './Components/Locations';
 
-function App() {
-  return (
-    <div className="app">
-      <main>
-        <h1>Hello, world!</h1>
-      </main>
-    </div>
-  );
-}
+class App extends Component {
+
```

Comment:
Watch the extra whitespace here. (Lines 12, 16, and 21 don't need to be blank lines!) Sounds like a minor quibble, but your future bosses will thank you if they don't have to clean up after you. : )